grapher/barCharts/DiscreteBarChart.tsx

```diff
@@ -616,24 +615,26 @@ export class DiscreteBarChart
         return sortedSeries
     }
 
-    @computed private get colorScheme(): ColorScheme | undefined {
-        // If this DiscreteBarChart stems from a LineChart, we want to match its (default) color
-        // scheme OWID Distinct. Otherwise, use an all-blue color scheme (`undefined`) as default.
-        const defaultColorScheme = this.manager.isLineChart
-            ? ColorSchemes["owid-distinct"]
-            : undefined
+    @computed private get colorScheme(): ColorScheme {
+        // We used to choose owid-distinct here as the default if this is a collapsed line chart but
+        // as part of the color revamp in Autumn 2022 we decided to make bar charts always default to
+        // an all-blue color scheme (singleColorDenim).
+        const defaultColorScheme =
+            ColorSchemes[ColorSchemeName.SingleColorDenim]
 
         return (
-            (this.manager.baseColorScheme
+            (this.manager.isLineChart
+                ? defaultColorScheme
+                : this.manager.baseColorScheme
                 ? ColorSchemes[this.manager.baseColorScheme]
                 : undefined) ?? defaultColorScheme
```

Comment:
I find nested ternaries with nullish coalescing pretty hard to reason about quickly.

Maybe it needs to be this way because of TypeScript but is there any chance we could do something more like this?

```
const defaultColorScheme = ColorSchemeName.SingleColorDenim
const colorScheme = this.manager.baseColorScheme || defaultColorScheme
return this.manager.isLineChart ? ColorSchemes[defaultColorScheme] : ColorSchemes[colorScheme]
```